client/my-sites/site-settings/jetpack-sync-panel/index.jsx

```diff
@@ -60,32 +62,56 @@ const JetpackSyncPanel = React.createClass( {
 		this.props.scheduleJetpackFullysync( this.props.siteId );
 	},
 
+	onClickDebug() {
+		debug( 'Clicked check connection button' );
```

Comment:
I know this method is not currently very useful. I will add analytics to it in a follow-up PR.